x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_model_health.tsx

```diff
@@ -100,11 +112,13 @@ const modelNotDeployedTooltip = i18n.translate(
 export interface TrainedModelHealthProps {
   modelState: TrainedModelState | MlModelDeploymentState;
   modelStateReason?: string;
+  isDownloadable?: boolean;
 }
 
 export const TrainedModelHealth: React.FC<TrainedModelHealthProps> = ({
   modelState,
   modelStateReason,
+  isDownloadable,
```

Comment:
Technically there's no difference between a non-deployed 3rd party model and a non-downloaded curated model (e.g. ELSER). But in the model details panel we're adding here we need to distinguish between these states, hence the `isDownloadable` flag.

This is a backward compatible change with other components using `TrainedModelHealth`; in those parts the label reads "Not started" for any model in this state.